tmc-plugin/src/fi/helsinki/cs/tmc/runners/TestRunHandler.java

```diff
@@ -78,40 +87,16 @@ public void bgTaskCancelled() {
         }
     }
 
-    private void startRunningTests(ExerciseRunner runner, final TmcProjectInfo projectInfo, final ResultCollector resultCollector) {
-        BgTask.start("Running tests", runner.getTestRunningTask(projectInfo), new BgTaskListener<TestRunResult>() {
-            @Override
-            public void bgTaskReady(TestRunResult result) {
-                Exercise ex = projectMediator.tryGetExerciseForProject(projectInfo, courseDb);
-                boolean canSubmit = ex.isReturnable();
-                resultDisplayer.showLocalRunResult(result.getTestCaseResults(), canSubmit, new Runnable() {
-                    @Override
-                    public void run() {
-                        exerciseSubmitter.performAction(projectInfo.getProject());
-                    }
-                }, resultCollector);
-            }
-
-            @Override
-            public void bgTaskFailed(Throwable ex) {
-                log.log(INFO, "StartRunningTests failed message: {0}, trace: {1}",
-                        new Object[] {ex.getMessage(), Arrays.deepToString(ex.getStackTrace())});
-                dialogDisplayer.displayError("Failed to run the tests: " + ex.getMessage());
-            }
-
-            @Override
-            public void bgTaskCancelled() {
-            }
-        });
-    }
-
     private AbstractExerciseRunner getRunner(TmcProjectInfo projectInfo) {
         switch (projectInfo.getProjectType()) {
             case JAVA_MAVEN:
+                log.log(INFO, "Maven task selected");
                 return new MavenExerciseRunner();
             case JAVA_SIMPLE:
+                log.log(INFO, "Ant task selected");
                 return new AntExerciseRunner();
             case MAKEFILE:
+                log.log(INFO, "Makefile task selected");
```

Comment:
These logs seem unnecessary (and misleading - this is not called when a user "selects" a project). Remove?